tests/run-tests.sh

```diff
@@ -48,7 +48,7 @@ check_config_files() {
   fi
 }
 
-setup_dss_binary() {
+setup_css_binary() {
   # One can setup environment variable DOCKER_STORAGE_SETUP to override
   # which binary is used for tests.
   if [ -n "$DOCKER_STORAGE_SETUP" ];then
```

Comment:
So you are retaining this for backward compatibility of running tests? I guess not a bad idea. We should probably also parse CONTAINER_STORAGE_SETUP and ask users to set that instead.

Or we could start emiting warning that DOCKER_STORAGE_SETUP environment variable has been deprecated. Use CONTAINER_STORAGE_SETUP instead.